pagopa/io-app
# [IASV-33, IASV-8] Show voucher generated screen

## Short description
This PR add the voucher generated screen which shows the voucher details after the voucher request succeed.
It also manages the timeout response, showing the associated message.

## List of changes proposed in this pull request
- Added the logic in `VoucherGeneratedScreen`
- Added type checking utils
- Updated the `SvGeneratedVoucherTimeoutScreen` image


https://user-images.githubusercontent.com/11773070/134685645-fa915e67-4c0d-42d3-9f1a-468c42b79b41.mp4

<img src="https://user-images.githubusercontent.com/11773070/134687714-639a99d2-3674-4c76-ab55-62499ac91b33.png" width=300>


## How to test
Starts the flow from the SiciliaVola service detail press the Voucher Generation button then complete the flow.
As category select "Severely disabled" if you want to test the happy path because for the other categories the destination informations are required and this screen is still to be developed.

Note that the details data are stubbed in the api call handler so you receive always the same response.
